bindata/network/kuryr/006-controller.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: kuryr-controller
+  namespace: kuryr
+  annotations:
+    kubernetes.io/description: |
+      This deployment launches the kuryr-controller component.
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: kuryr-controller
+      labels:
+        app: kuryr
+        component: network
+        type: infra
+        openshift.io/component: network
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      serviceAccountName: kuryr
+      hostNetwork: true
+      priorityClassName: system-node-critical
+      containers:
+      - image: {{ .ControllerImage }}
+        name: controller
+{{ if (default true .ControllerEnableProbes) eq "true" }}
+        readinessProbe:
+          httpGet:
+            path: /ready
+            port: {{ default 8082 .ControllerProbesPort }}
+            scheme: HTTP
+          timeoutSeconds: 5
+        livenessProbe:
+          httpGet:
+            path: /alive
+            port: {{ default 8082 .ControllerProbesPort }}
+          initialDelaySeconds: 15
+{{ end }}
+        # FIXME(dulek): This shouldn't be required, but without it selinux is
```

Comment:
Now that this is a ConfigMap, I bet privileged isn't required anymore. Worth a try, but can be a followup.

Reply:
We had those selinux issues when that was a ConfigMap. I can totally try removing it and retest. Let's assume it's a follow up, it's trivial.